react/AppSections/components/SmallAppItem.jsx

```diff
@@ -61,7 +61,6 @@ export const SmallAppItem = ({
 }
 
 SmallAppItem.propTypes = {
-  t: PropTypes.func.isRequired,
```

Comment:
Why is the proptype removed?

Reply:
I don't know, I guess it was to remove a warning during tests. Since it is not part of the PR, I removed the removal.